### PR TITLE
Conscious language blacklist 1

### PR DESCRIPTION
**Pull request by rvykydal**
Note: the main reason for picking blocklist/passlist (not denylist/allowlist) was consistency with RHEL 8 installation documentation where it is already used.

### CODE REVIEW

File: pyanaconda/modules/payloads/base/utils.py [hunk 1]
```diff
@@ -35,19 +35,19 @@ def create_root_dir(sysroot):
...
     Copy modules from modprobe.blacklist=<module> on cmdline to
     /etc/modprobe.d/anaconda-blacklist.conf so that modules will
-    continue to be blacklisted when the system boots.
+    continue to be blocklisted when the system boots.
     """
     if "modprobe.blacklist" not in kernel_arguments:
         return
 
     mkdirChain(os.path.join(sysroot, "etc/modprobe.d"))
     with open(os.path.join(sysroot, "etc/modprobe.d/anaconda-blacklist.conf"), "w") as f:
```
Comment (poncovka) [inline pyanaconda/modules/payloads/base/utils.py:49]: Don't we want to change the name of the file as well, or is it documented somewhere?

**Review by poncovka — APPROVED** [review]
It looks good to me. Thanks!

**Comment by poncovka** [comment]
Do we want to do this in Fedora 34? Isn't it too late?

File: pyanaconda/modules/payloads/base/utils.py (hunk 1 above)
Comment (poncovka) [reply pyanaconda/modules/payloads/base/utils.py:49]: Never mind. I have found https://github.com/rhinstaller/anaconda/pull/3203.

**Review by VladimirSlavik — APPROVED** [review]
I think this is mostly fine. The word "blocklist" makes me really sad, but if that's used elsewhere too, it's better to stay consistent.

**Review by VladimirSlavik — CHANGES_REQUESTED** [review]
On a further probing, I hear kernel dev wants to use "denylist". I think we should stay consistent with that.

I think it's actually an issue of RHEL docs choosing something else for installer.

**Comment by rvykydal** [comment]
@VladimirSlavik thank you for digging into it, I pushed new version using 'denylist'

**Review by poncovka — APPROVED** [review]

**Review by poncovka — APPROVED** [review]

**Review by VladimirSlavik — APPROVED** [review]

**Pull request merged**